{
	"id": "oval:org.almalinux.alsa:def:20240464",
	"version": "635",
	"class": "patch",
	"metadata": {
		"Title": "ALSA-2024:0464: python-urllib3 security update (Moderate)",
		"reference": [
			{
				"ref_id": "ALSA-2024:0464",
				"source": "ALSA",
				"ref_url": "https://errata.almalinux.org/9/ALSA-2024-0464.html"
			},
			{
				"ref_id": "CVE-2023-43804",
				"source": "CVE",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2023-43804"
			},
			{
				"ref_id": "CVE-2023-45803",
				"source": "CVE",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2023-45803"
			},
			{
				"ref_id": "ALSA-2024:0464",
				"source": "ALSA",
				"ref_url": "https://errata.almalinux.org/9/ALSA-2024-0464.html"
			}
		],
		"Description": "The python-urllib3 package provides the Python HTTP module with connection pooling and file POST abilities.\n\nSecurity Fix(es):\n\n* python-urllib3: Cookie request header isn't stripped during cross-origin redirects (CVE-2023-43804)\n* urllib3: Request body not stripped after redirect from 303 status changes request method to GET (CVE-2023-45803)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
		"advisory": {
			"from": "packager@almalinux.org",
			"Severity": "Moderate",
			"Rights": "Copyright 2024 AlmaLinux OS",
			"issued": {
				"date": "2024-01-25"
			},
			"updated": {
				"date": "2024-01-25"
			},
			"affected_cpe_list": {
				"Cpe": [
					"cpe:/a:almalinux:almalinux:9",
					"cpe:/a:almalinux:almalinux:9::appstream",
					"cpe:/a:almalinux:almalinux:9::crb",
					"cpe:/a:almalinux:almalinux:9::highavailability",
					"cpe:/a:almalinux:almalinux:9::nfv",
					"cpe:/a:almalinux:almalinux:9::realtime",
					"cpe:/a:almalinux:almalinux:9::resilientstorage",
					"cpe:/a:almalinux:almalinux:9::sap",
					"cpe:/a:almalinux:almalinux:9::sap_hana",
					"cpe:/a:almalinux:almalinux:9::supplementary",
					"cpe:/o:almalinux:almalinux:9",
					"cpe:/o:almalinux:almalinux:9::baseos"
				]
			},
			"bugzilla": [
				{
					"href": "https://bugzilla.redhat.com/2242493",
					"id": "2242493"
				},
				{
					"href": "https://bugzilla.redhat.com/2246840",
					"id": "2246840"
				}
			],
			"cve": [
				{
					"text": "CVE-2023-43804",
					"public": "20231004",
					"href": "https://access.redhat.com/security/cve/CVE-2023-43804",
					"impact": "Moderate",
					"cwe": "CWE-200",
					"cvss3": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:N"
				},
				{
					"text": "CVE-2023-45803",
					"public": "20231013",
					"href": "https://access.redhat.com/security/cve/CVE-2023-45803",
					"impact": "Moderate",
					"cwe": "CWE-200",
					"cvss3": "CVSS:3.1/AV:A/AC:H/PR:H/UI:N/S:U/C:H/I:N/A:N"
				}
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterions": [
					{
						"test_ref": "oval:org.almalinux.alba:tst:20225749023",
						"comment": "AlmaLinux 9 is installed"
					},
					{
						"test_ref": "oval:org.almalinux.alsa:tst:20240464001",
						"comment": "python3-urllib3 is earlier than 0:1.26.5-3.el9_3.1"
					},
					{
						"test_ref": "oval:org.almalinux.alsa:tst:20240464002",
						"comment": "python3-urllib3 is signed with AlmaLinux OS 9 key"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.almalinux.alba:tst:20225749024",
				"comment": "AlmaLinux must be installed"
			}
		]
	}
}
